src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,97 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+import { NavLink, useParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { Loader } from './Loader';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>();
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+  const { slug = '' } = useParams();
+
+  useEffect(() => {
+    setIsLoading(true);
```

Comment:
redundant - it's initial state value

Reply:
https://github.com/mate-academy/react_people-table-basics/pull/869/files#r1272546705
looks like not fixed